src/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -797,7 +786,7 @@
     <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.SetLastError.cs">
       <Link>Interop\Windows\mincore\Interop.SetLastError.cs</Link>
     </Compile>
-    <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
+    <Compile Condition="'$(EnableDummyGlobalizationImplementation)' != 'true'" Include="..\..\Common\src\Interop\Windows\mincore\Interop.Globalization.cs">
       <Link>Interop\Windows\mincore\Interop.Globalization.cs</Link>
     </Compile>
     <Compile Include="..\..\Common\src\Interop\Windows\mincore\Interop.Idna.cs">
```

Comment:
`Interop.Idna.cs` and `Interop.Normalization.cs` is also part of globalization - move them too.